Prioritize venv with conda as secondary option in virtual env docs (#3686)
kedro-org/kedro: docs/source/get_started/install.md

## Create a virtual environment for your Kedro project

We strongly recommend utilizing `venv` as your virtual environment manager if you don't already use it.

``` {tip}
[Read more about virtual environments for Python projects](https://realpython.com/python-virtual-environments-a-primer/) or [watch an explainer video about them](https://youtu.be/YKfAwIItO7M).
```

### How to create a new virtual environment using `venv`

> The recommended approach. If you use Python 3, you should already have the `venv` module installed with the standard library. Create a directory for working with your project and navigate to it. For example:

Reviewer: Is it missing some venv activate script? I expect something like `bin/activate`

Author: it is provided, I haven't change that part:
```
Activate this virtual environment:
source .venv/bin/activate # macOS / Linux
```

## Create a virtual environment for your Kedro project

> We strongly recommend utilizing `venv` as your virtual environment manager if you don't already use it.

Reviewer 2: ```suggestion
We strongly recommend using `venv` as your virtual environment manager if you don't already use it.
```

("utilising" sounds unnecessarily complex?)

```bash
deactivate
```

> 

Author: I agree, fixed

> ### How to create a new virtual environment without using `venv`

Reviewer 2: "without `venv`" implies that we're covering more options. I would just rename this to

```suggestion
### How to create a new virtual environment using `conda`
```

>  [The second, less popular option is to install Conda.](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) Than from your terminal:

Reviewer 2: Maybe

```suggestion
 [Another popular option is to use Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/). After you install it, execute this from your terminal:
```

>  [Another popular option is to use Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/). After you install it, execute this from your terminal:

Reviewer 2: ```suggestion
[Another popular option is to use Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/). After you install it, execute this from your terminal:
```

```bash
conda create --name kedro-environment python=3.10 -y
```

The example below uses Python 3.10, and creates a virtual environment called `kedro-environment`. You can opt for a different version of Python (any version >= 3.8 and <3.12) for your project, and you can name it anything you choose.

> The `conda` virtual environment is not dependent on your current working directory and can be activated from any directory:

Reviewer: Is this conda or venv? I see conda instruction under venv section

Author: maybe you should better have a look to final version on https://github.com/kedro-org/kedro/blob/make-pip-and-venv-installation-instructions-more-visible/docs/source/get_started/install.md not for a code difference, looks like it's hard to understand here. I see it's all right with sections
